docs/source/application_development/porter.rst

```diff
@@ -466,116 +466,6 @@ Example Response
     }
 
 
-POST /publish_treasure_map
```

Comment:
😢

Reply:
Sorry to rug pull ya' on this one... It may return someday!